src/app/components/MediaLoader/Placeholder/index.tsx

```diff
@@ -30,31 +36,54 @@ const MediaPlayerPlaceholder = ({
     guidanceMessage,
   } = mediaInfo ?? {};
 
+  const playButton = (
+    <PlayButton
+      css={styles.playButton}
+      // eslint-disable-next-line @typescript-eslint/no-empty-function
+      onClick={() => {}}
+      title={title}
+      datetime={datetime}
+      duration={duration}
+      durationSpoken={durationSpoken}
+      type={type}
+      guidanceMessage={guidanceMessage}
+      className="focusIndicatorRemove"
+    />
+  );
+
+  const experimentPlayButton = (
```

Comment:
Can we use `transcript` instead of `experiment` in variable names? (Let's pretend it's not an experiment any more!)

`playButtonWithTranscript` or something along those lines?